apps/web/src/views/Mev/hooks/index.ts

```diff
@@ -117,9 +117,16 @@ export function useIsMEVEnabled() {
 
 export const useShouldShowMEVToggle = () => {
   const { walletSupportsAddEthereumChain, isLoading: isWalletSupportLoading } = useWalletSupportsAddEthereumChain()
-  const { account } = useActiveWeb3React()
+  const { account, chainId } = useActiveWeb3React()
   const { isMEVEnabled, isLoading } = useIsMEVEnabled()
-  return !isMEVEnabled && !isLoading && !isWalletSupportLoading && Boolean(account) && walletSupportsAddEthereumChain
+  return (
+    !isMEVEnabled &&
+    !isLoading &&
+    !isWalletSupportLoading &&
+    Boolean(account) &&
+    walletSupportsAddEthereumChain &&
+    chainId === ChainId.BSC
```

Comment:
Shall we move this part into the `useIsMEVEnabled` hook? For example expose a variable called `isMEVProtectAvailable`?

Reply:
done